javascripts/discourse/components/home-list.gjs

```diff
@@ -159,13 +190,64 @@ export default class HomeList extends Component {
           </li>
         {{/each}}
         <LoadMore @selector=".discover-list__item" @action={{this.loadMore}} />
+        {{#if this.homepageFilter.loading}}
+          <li class="discover-list__item --loading">
+            <ConditionalLoadingSpinner
+              @condition={{this.homepageFilter.loading}}
+            />
+          </li>
+        {{/if}}
+        {{#unless this.homepageFilter.loading}}
+          {{#if this.multiPageEnd}}
+            <li class="discover-list__item --end">
+              <svg
```

Comment:
An svg like this a good target for extracting to a separate (template-only) component